### PR TITLE
Added Homepage and search-sort feature 

### PR DESCRIPTION
**Pull request by nikhil-sharma-nks**
### 4. sort and filter
chrome-capture-2022-4-24

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -17,29 +17,39 @@
...
   --text-color: #ffffff;
-  --text-color-secondary: #aaaaaa;
-  --search-box-background-color: hsl(0, 0%, 7%);
-  --search-box-text-color: hsla(0, 100%, 100%, 0.88);
-  --border-color: hsl(0, 0%, 18.82%);
-  --drawer-item-hover: hsl(0, 0%, 20%);
-  --cateogry-pills-background-color: rgba(255, 255, 255, 0.1);
-  --cateogry-pills-hover: rgba(255, 255, 255, 0.2);
+  --background-color-secondary: hsl(0, 0%, 37%);
+  --category-pills-background-color: rgba(255, 255, 255, 0.1);
```
Comment (HarshnaNHaswani) [inline src/App.scss:32]: use same format for colors

File: src/routes/RoutesContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const RoutesContainer = () => {
+  return (
+    <Routes>
+      <Route path='/' element={<Home />} />
+      <Route path='/home' element={<Home />} />
+      <Route path='/mock' element={<Mockman />} />
+    </Routes>
+  );
+};
```
Comment (HarshnaNHaswani) [inline src/routes/RoutesContainer.jsx]: suggestion: use useRoutes() hook instead

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  //     type: 'LOGOUT',
+  //   });
+  //   productDispatch({ type: 'LOGOUT' });
+  //   navigate('/');
+  // };
+  const handleFocus = () => {
+    // if (location.pathname === '/') {
+    //   navigate('/');
+    // }
+  };
```
Comment (HarshnaNHaswani) [inline src/components/Navbar/Navbar.jsx]: remove commented code before committing

**Review by HarshnaNHaswani — COMMENTED** [review]
few suggestions, otherwise good to go.
- nice folder structure
- clean code
- easy to understand the data flow

File: src/routes/RoutesContainer.jsx (hunk 2 above)
Comment (nikhil-sharma-nks) [reply src/routes/RoutesContainer.jsx]: thanks for the suggestion

File: src/App.scss (hunk 1 above)
Comment (atul-gairola) [inline src/App.scss:32]: You could use same format of colors

File: src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  // const { authState, authDispatch } = useAuth();
+  const { searchQuery } = videoState;
+  // const handleLogout = () => {
+  //   makeToast('You Are Now Logged Out', 'success');
+  //   authDispatch({
+  //     type: 'LOGOUT',
+  //   });
+  //   productDispatch({ type: 'LOGOUT' });
+  //   navigate('/');
+  // };
```
Comment (atul-gairola) [inline src/components/Navbar/Navbar.jsx]: remove commented code if it is not neccessary

**Review by atul-gairola — APPROVED** [review]
UI looks good just small changes

**Pull request merged**